ee/server/service/mdm.go

```diff
@@ -871,6 +907,9 @@ func (svc *Service) mdmSSOHandleCallbackAuth(
 		return "", idpAcc.UUID, eulaToken, originalURL, nil
 	}
 
+	// TODO(mna): is this correct to get the "automatic" enrollment profile for BYOD?
+	// I think we may want some "manual" one if it exists? Or none at all?
```

Comment:
THis is irrelevant in the case of BYOD, it's the OTA enrollment profile that will get deployed, the `depProf` returned here is unused in the case of BYOD (the next step is the download OTA profile, and it doesn't use the `profile_token` query string that represents the `depProf.Token`).